README.md

```diff
@@ -18,34 +18,56 @@ Apollos is a reactive application framework for building high speed, web + nativ
 * <a href="https://www.biblegateway.com/passage/?search=1%20Corinthians%203:6&version=NIV">1 Corinthians 3:6</a>
 * <a href="https://www.biblegateway.com/passage/?search=Titus%203:13&version=NIV">Titus 3:13</a>
 
-## Structure
+## Prerequisites
 
-This repo contains the core Apollos framework, as well as the sites that share the core Apollos code:
+- [Meteor](curl https://install.meteor.com/ | sh): `curl https://install.meteor.com/ | sh`;
+- [Node 6](https://nodejs.org/en/download/)
 
-- `/apollos`: shared code
-- `/sites`: root site directory
-  - `/newspring`: newspring.cc (currently my.newspring.cc)
+## Quick Start
 
-## Prerequisites
+```
+git clone git@github.com:NewSpring/Apollos.git
+cd Apollos
+npm link
```

Comment:
@jbaxleyiii it does! i tested it :). figured it's one less command for people to run initally